modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase

```diff
@@ -53,6 +53,26 @@ definition {
 			value = "Test Test");
 	}
 
+	@description = "Story: LPS-167803 | Test Summary: Verify that it is possible to search for a project with just the project code"
+	@priority = 2
+	test AssertThatAbleToSearchWithProjectCode {
+		task ("Given the User go to Customer Portal Site") {
+			ApplicationsMenu.gotoSite(key_site = ${testSiteName});
+		}
+
+		task ("When search for a project by the ERC") {
```

Comment:
"When the User searches for a project by ERC"

Reply:
Okay